extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -0,0 +1,260 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id":
+    "https://ns.adobe.com/xdm-extensions/adobe/experience/campaign/experienceevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Campaign ExperienceEvent Extension",
+  "type": "object",
+  "meta:extends": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Adobe Campaign extension to ExperienceEvent.",
+  "definitions": {
+    "experienceevent": {
+      "properties": {
+        "https://ns.adobe.com/experience/campaign/message/id": {
+          "title": "Message Identifier",
+          "type": "number",
+          "description": "Adobe Campaign message's broadlog ID."
```

Comment:
What's a "broadlog"?

Reply:
a unique broadlog refers to every single message sent as part of a campaign activity (which is called a delivery). I'd rather call it a  `message` here.